src/commands/base.ts

```diff
@@ -51,13 +51,19 @@ export default abstract class Base extends Command {
             description: 'Disable all interactive flows and format output for easy parsing.',
             helpGroup: 'Global'
         }),
+        'caller': Flags.enum({
+            description: 'The integration that is calling the CLI.',
+            helpGroup: 'Global',
+            options: ['github', 'bitbucket', 'cli']
+        }),
     }
 
     token = ''
     projectKey = ''
     authPath = path.join(this.config.configDir, 'auth.yml')
     configPath = path.join(this.config.configDir, 'user.yml')
     repoConfigPath = '.devcycle/config.yml'
+    caller = ''
```

Comment:
This should either be `undefined` or default to `cli`. `setDVCReferrer` won't default to `cli` if caller is passed as an empty string (the default param is only used if the passed param isn't set)

Reply:
updated to default to `cli`